src/math/noise.js

```diff
@@ -255,12 +278,12 @@ p5.prototype.noise = function(x, y = 0, z = 0) {
  * Adjusts the character of the noise produced by the
  * <a href="#/p5/noise">noise()</a> function.
  *
- * Perlin noise values are computed over several octaves, similar to harmonics
- * in music. Lower octaves contribute more to the output signal. They define
- * the overall intensity of the noise. Higher octaves create finer-grained
- * details.
+ * Perlin noise values are created by adding layers of noise together. The
+ * noise layers, called octaves, are similar to harmonics in music. Lower
+ * octaves contribute more to the output signal. They define the overall
+ * intensity of the noise. Higher octaves create finer-grained details.
  *
- * By default, noise values are computed over four octaves. Each higher octave
+ * By default, noise values combine four octaves. Each higher octave
```

Comment:
Looks great! Just a small wording suggestion: "By default, noise values are created by combining four octaves" might be clearer

Reply:
Done and done!